app/RDF/GraphEditor.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\RDF;
+
+use EasyRdf\Graph;
+use EasyRdf\Literal;
+use EasyRdf\Resource;
+
+final class GraphEditor
+{
+    private Graph $graph;
+
+    private const TYPE_IDENTIFICATOR = 'adms:Identifier';
+
+    private const PROPERTY_IDENTIFICATOR = 'adms:identifier';
+
+    private const PROPERTY_AANGEMAAKT_OP = 'dcterms:created';
+    private const PROPERTY_LAATST_AANGEPAST = 'dcterms:modified';
+
+    private const PROPERTY_IDENTIFICATOR_NOTATION = 'skos:notation';
+    private const PROPERTY_IDENTIFICATOR_TOEGEKEND_DOOR = 'dcterms:creator';
+    private const PROPERTY_IDENTIFICATOR_TOEGEKEND_DOOR_AGENT = 'https://fixme.com/example/dataprovider/publiq';
+    private const PROPERTY_IDENTIFICATOR_NAAMRUIMTE = 'generiek:naamruimte';
+    private const PROPERTY_IDENTIFICATOR_LOKALE_IDENTIFICATOR = 'generiek:lokaleIdentificator';
+    private const PROPERTY_IDENTIFICATOR_VERSIE_ID = 'generiek:versieIdentificator';
+
+    private const PROPERTY_WORKFLOW_STATUS = 'udb:workflowStatus';
+    private const PROPERTY_WORKFLOW_STATUS_DRAFT = 'https://data.publiq.be/concepts/workflowStatus/draft';
+
+    private function __construct(Graph $graph)
+    {
+        $this->graph = $graph;
+    }
+
+    public static function for(Graph $graph): self
+    {
+        return new self($graph);
+    }
+
+    public function setGeneralProperties(
+        string $uri,
```

Comment:
Nit: Maybe using `$subject` or `$resource` instead of `$uri` might be a little bit clearer here, because the `$type` for example will also be a URI. And same in the other methods where for example `$property` will also contain a URI.